NEWS.md

```diff
@@ -1,11 +1,19 @@
 # brms 2.16.3++
 
+### Other changes
+
+* Argument `kfold_seed` has been added to `get_refmodel.brmsfit()`. (#1286)
```

Comment:
Is it necessary to add two different seed arguments, or could we get away with one somehow?

Reply:
Sure, we could have a single seed argument (perhaps called `brms_seed`?) which is then used at the beginning of `get_refmodel.brmsfit()` to infer separate seeds for `kfold()` and `ref_predfun`. Do you want me to push such a modification?

Reply:
Is there a reason not to call the argument `seed`, or would that clash with somthing else?

Reply:
Yes, that would clash with argument `seed` of projpred's functions `varsel()`, `cv_varsel()`, and `project()` which all use their ellipsis to pass arguments to `get_refmodel()`.

Reply:
I see. Yes, `brms_seed` would be sensible then. Thank you for working on this PR!

Reply:
Just pushed commit 97c637e (as well as f8d9d26) which performs this replacement.